docs/plugins/rcpt_to.routes.md

```diff
@@ -1,4 +1,4 @@
-# `rcpt_to.routes`
+# rcpt_to.routes
```

Comment:
Yeah the regexp is horribly buggy in the site builder:

```
    my ($title) = ($output =~ /<h1>([^<]*)/);
    $title ||= "Haraka";
```

So "# `plugin.name`" would get turned into <h1><code>plugin.name</code></h1>
and thus break the regexp. Gross.

I've asked for help on the site builder before but nobody comes forwards :(

Matt.

On Thu, Aug 18, 2016 at 12:24 PM, Aaron C. de Bruyn <
notifications@github.com> wrote:

> In docs/plugins/rcpt_to.routes.md
> <https://github.com/haraka/Haraka/pull/1568#discussion_r75340896>:
>
> > @@ -1,4 +1,4 @@
> > -# `rcpt_to.routes`
> > +# rcpt_to.routes
>
> I didn't test it. I looked at several of the other docs and assumed that
> was why it showed up in the plugins list as 'Haraka' instead of
> 'rcpt_to.routes'. If that's not where the website generates the name, let
> me know and I'll change it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/haraka/Haraka/pull/1568/files/ec5b41f392d310c8df0401503eafc74663754164#r75340896>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAobY2ymEHoJtTT9hRz6kJosSFNVxHFvks5qhIc_gaJpZM4Jmyno>
> .
>